forge-gui/res/cardsfolder/upcoming/bortuk_bonerattle.txt

```diff
@@ -0,0 +1,9 @@
+Name:Bortuk Bonerattle
+ManaCost:4 B G
+Types:Legendary Creature Troll Shaman
+PT:4/4
+T:Mode$ ChangesZone | ValidCard$ Card.wasCastByYou+Self | Destination$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ Domain — When CARDNAME enters the battlefield, if you cast it, choose target creature card in your graveyard. Return that card to the battlefield if its mana value is less than or equal to the number of basic land types among lands you control. Otherwise, put it into your hand.
+SVar:TrigChangeZone:DB$ ChangeZone | ValidTgts$ Creature.YouOwn | Origin$ Graveyard | Destination$ Battlefield | ConditionDefined$ Targeted | ConditionPresent$ Card.cmcLEX | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ Targeted | Origin$ Graveyard | Destination$ Hand | ConditionDefined$ Targeted | ConditionPresent$ Card.cmcGTX
+SVar:X:Count$Domain
+Oracle:Domain — When Bortuk Bonerattle enters the battlefield, if you cast it, choose target creature card in your graveyard. Return that card to the battlefield if its mana value is less than or equal to the number of basic land types among lands you control. Otherwise, put it into your hand.
```

Comment:
DeckHas:Ability$Graveyard
I like the RemoveDeck:Random for these Domain cards since most decks don't pack in too many basic land types – isn't it an old mechanic? Was there some other AI in place for it?

Reply:
Added Graveyard flag.

I did post a question to the Discord about what to do with AI flags for domain cards. Aven Trailblazer, an old Domain card, has this param: `SVar:BuffedBy:Plains,Island,Swamp,Mountain,Forest`. Should we be including this tag for all Domain cards from here on out? Is it more efficient to just restrict AI playability with `RemoveDeck:Random`?

Reply:
Made changes as advised by Agetian in the discord